book/src/hints.md

```diff
@@ -30,10 +31,13 @@ Leading zeros are supported, for example
 ``` rust
 # extern crate defmt;
 defmt::info!("{=u8:03}", 42); // -> INFO 042
-defmt::info!("{=u8:08X}", 42); // -> INFO 0x0000002A
+defmt::info!("{=u8:08X}", 42); // -> INFO 0000002A
+defmt::info!("{=u8:#010X}", 42); // -> INFO 0x0000002A
 ```
 
-No further customization like padding is supported (at the moment).
+When the alternate form is used for hex and binary, the `0x`/`0b` length is subtracted from the
+leading zeros.  This matches [`core::fmt` behavior](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=b11809759f975e266251f7968e542756). No further
+customization like padding is supported (at the moment).
```

Comment:
> No further customization like padding is supported (at the moment).

Aren't the leading zeroes a kind of padding?

Reply:
Hmm good point, would you like me to change this? Maybe to `No further customization is supported (at the moment).`?

Reply:
Yes :)

Reply:
(I guess this should have changed with your last PR, but good that we caught it now ;D)